### ISSUE TITLE
[Accessibility] [Nav Tabs] [for all navs tab]> Navigation region lacks an accessible name

### ISSUE THREAD

**Issue by silvalaura** [cengage/react-magma#2021]
[A11Y-616](https://jira.cengage.com/browse/A11Y-616)

**Environment:**
Windows 11 / Chrome 137 / JAWS 2025
Windows 11 / Edge 136 / NVDA 2025
MacOS 15 / Safari / VoiceOver

**Steps to Reproduce:**
1. Open the URL: https://react-magma.cengage.com/version/4.9.1/api/nav-tabs/#
2. Turn on a screen reader (JAWS/NVDA/VoiceOver).
3. Navigate to the 'for all navs tab' subcomponent.
4. Listen to the screen reader announcement.

**Actual Result:**
The `<nav>` element containing the links does not have an accessible name. As a result, screen readers announce it generically as "navigation" region without context.

Refer to the screenshot:
https://app.screencast.com/PASMOnCYHPbbp
https://app.screencast.com/0hXfWU8K8PLT7

**Expected Result:**
The `<nav>` element should have an accessible name that describes its purpose, such as "For All Navs Tab". Screen reader should announce as "For all navs tab navigation region".

**User Impact:**
Screen reader users may not understand the purpose of the navigation region, especially if there are multiple `<nav>` elements on the page. This can lead to confusion and inefficient navigation.

WCAG Guidelines:
1.3.1 Info and Relationships (Level A)

**Bug Fix Recommendation:**
Add an accessible name to the `<nav>` element using aria-labelledby

**Comment by silvalaura** [comment]
Solution: in our examples with multiple navtabs components, use `aria-labelledby`. Also note in the docs that aria-labelledby is required when there are multiple `<nav>`s on a page.

**Comment by silvalaura** [comment]
> We have reverified this issue on URL: [React Magma](https://next--upbeat-sinoussi-f675aa.netlify.app/) and observed that it is still not resolved. The aria-labelledby attribute used on the <nav> element is not implemented correctly. This attribute should reference an existing element in the DOM using an id. If a direct label is intended, aria-label should be used instead. Currently, the referenced id is missing.
> 
> Screencast: https://app.screencast.com/nVYNDxB0mf30D
